### PR TITLE
feat: Added skipLocalhost option

### PR DESCRIPTION
**Pull request by alexandrtovmach**
Implemented `skipLocalhost` option with tests and README.

Related issue #16

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Functionality looks good to me. Great tests!

File: test.js [hunk 1]
```diff
@@ -164,4 +174,50 @@ describe('remark-lint-no-dead-urls', () => {
       expect(vFile.messages.length).toBe(0);
     });
   });
+
+  describe('skipLocalhost: true', () => {
+    test('localhost', () => {
+      const lint = processMarkdown(
+        dedent`
+							- [http://localhost](http://localhost)
```
Comment (davidtheclark) [inline test.js]: This indentation looks pretty extreme 😬

File: README.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ All options are optional. The options object may contain any of the following pr
 - **skipOffline** `{boolean}` - Default: `false`.
   By default, if you are offline when you run the check you will receive a warning.
   If you want to let offline runs quietly pass, set this option to `true`.
+- **skipLocalhost** `{boolean}` - Default: `false`.
+  By default, all local links testing to a live status, and if project not running locally you'll receive a warning.
```
Comment (davidtheclark) [inline README.md]:
```suggestion
  By default, `localhost` links are treated the same as other links, so if your project is not funning locally you'll receive a warning.
```

File: README.md [hunk 3]
```diff
@@ -35,6 +35,9 @@ All options are optional. The options object may contain any of the following pr
...
+  If you want to skip localhost links (http://localhost, http://127.0.0.1), set this option to `true`.
```
Comment (davidtheclark) [inline README.md]:
```suggestion
  If you want to ignore `localhost` links (e.g. `http://localhost/*`, `http://127.0.0.1/*`), set this option to `true`.
```

File: index.js [hunk 4]
```diff
@@ -11,6 +11,11 @@ function noDeadUrls(ast, file, options) {
   const aggregate = (node) => {
     const url = node.url;
     if (!url) return;
+    if (
+      options.skipLocalhost &&
+      /(https?:\/\/)(localhost|127\.0\.0\.1)(:\d+)?/.test(url)
```
Comment (davidtheclark) [inline index.js]: Should this regexp start with a `^`?

**Comment by davidtheclark** [comment]
Thanks, @alexandrtovmach. I had just a couple of suggestions and questions. We can take care of those then merge & release.

File: test.js (hunk 1 above)
Comment (alexandrtovmach) [reply test.js]: Ooops, that's prettier config with tabs))

File: README.md (hunk 2 above)
Comment (alexandrtovmach) [reply README.md]: _funning_ should be _running_, right?

**Comment by alexandrtovmach** [comment]
@davidtheclark Updated, please take a look

**Review by davidtheclark — APPROVED** [review]

**Comment by davidtheclark** [comment]
Published in 1.0.0.

**Pull request merged**